app/views/exam_templates/_form.html.erb

```diff
@@ -22,6 +22,7 @@
       <p><%= raw t('assignment.required_fields') %> <span class='required_field'>*</span></p>
       <%= f.fields_for :template_divisions do |form| %>
         <p>
+          <%= form.check_box :_destroy%>
```

Comment:
There should be a space before the `%>`. Also, this checkbox should have a label.